old-index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
if this is an old file that is no longer being used I would not include it in production.

Reply:
Gotcha! Edited out.